src/common/utils/Comparator.ts

```diff
@@ -0,0 +1,25 @@
+import { Data } from '../../common/types/TableTypes.types';
+import { SortingDirection } from './SortingDirection';
+
+export function descendingComparator<T>(firstValue: T, secondValue: T, orderBy: keyof T) {
```

Comment:
`firstValue`? I think it is more like `recordA` and `recordB`
`orderBy`? I think it is more like `fieldName`